src/config.json

```diff
@@ -937,8 +937,10 @@
     "metapass.fyi"
   ],
   "blacklist": [
+    "restoremetadata.com",
+    "mywalletauthenticator.weebly.com",
     "sandboxgame.app",
-    "shibasociaiclub.com",
+    "shibasocialclub.com",
```

Comment:
@AlexHerman1 @Ester800 Was this a mistake? This change seems unrelated to the linked ZenDesk ticket. Also we got a report that this site was legitimate: #6337

Reply:
i believe the line that was removed is the phishing site, and the other is the legitimate site: shibasocialclub.com

Reply:
Fixed here: https://github.com/MetaMask/eth-phishing-detect/pull/6396